build.rs

```diff
@@ -102,6 +102,15 @@ fn build_fips() {
         .allowlist_item("EVP_.*")
         .allowlist_item("evp_.*")
         .allowlist_item("BN_.*")
+        .allowlist_item("d2i_ECPKParameters")
+        .allowlist_item("d2i_ECDSA_SIG")
+        .allowlist_item("i2d_ECDSA_SIG")
+        .allowlist_item("d2i_ASN1_OCTET_STRING")
+        .allowlist_item("i2d_ASN1_OCTET_STRING")
+        .allowlist_item("EC_GROUP_.*")
+        .allowlist_item("ECDSA_SIG_.*")
+        .allowlist_item("ASN1_STRING_.*")
+        .allowlist_item("NID_undef")
```

Comment:
You can probably allowlist "d2i_.\*" "i2d_.\*", is NID_undef really needed?

Reply:
Not necessarily. It can be removed and I can use just the number 0 or something.